sample_global_configs.json

```diff
@@ -10,6 +10,7 @@
     "TEMPLATEFLOW_DIR": "",
 
     "SESSIONS": ["1","2"],
+    "VISITS": ["V01", "V02"],
```

Comment:
I wonder if we should stick with integers here or actual `V01`, `V02` strings etc.. since for PPMI at least we will match the `sessions` and `visits`. Thoughts?

Reply:
I used different codes for visits just to show that they could be different, but if you want I can change it to be the same as sessions. I don't have a strong preference.

Reply:
Good point. Okay let's keep them different